.gitignore

```diff
@@ -12,3 +12,7 @@ pom.xml.asc
 ccw*
 /.idea
 /*.iml
+;; Clojure tooling
+.clj-kondo
```

Comment:
.clj-kondo folder shouldn't be completely ignored, config.edn and hook files should be included in the repository.

Same holds for .lsp/config.edn in fact.

For Clj-kondo and LSP it is the `.cache` folder under their folders that should be ignored.

I'm not sure about Calva configuration.